src/test/java/org/apache/drillui/test/framework/testng/unsecure/storage/StoragePluginsCRUDTest.java

```diff
@@ -50,10 +50,6 @@
   @BeforeClass
   public void setupStoragePlugins() {
     RestBaseSteps.setupStoragePlugins();
-  }
-
-  @BeforeMethod
-  public void beforeMethod() {
```

Comment:
It's better to do cleanup before every test method.

Reply:
Done